server/routes/redirects.py

```diff
@@ -30,31 +30,35 @@ def kg():
         url = url_for('browser.kg_entity', dcid=dcid)
     else:
         url = url_for('browser.kg_main')
-    return redirect(url, code=302)
+    return redirect(url, code=302, _scheme='https')
 
 
 @bp.route('/gni')
 def gni():
-    return redirect(url_for('tools.timeline'), code=302)
+    return redirect(url_for('tools.timeline'), code=302, _scheme='https')
 
 
 @bp.route('/scatter')
 def scatter():
-    return redirect(url_for('tools.scatter'), code=302)
+    return redirect(url_for('tools.scatter'), code=302, _scheme='https')
 
 
 @bp.route('/documentation')
 def documentation():
-    return redirect('https://docs.datacommons.org/', code=302)
+    return redirect('https://docs.datacommons.org/', code=302, _scheme='https')
 
 
 @bp.route('/colab')
 def colab():
-    return redirect('https://docs.datacommons.org/tutorials.html', code=302)
+    return redirect('https://docs.datacommons.org/tutorials.html',
+                    code=302,
+                    _scheme='https')
 
 
 # This is used to handle explore more link from Google search. Do not remove.
+# arg params from search: mprop, dcid, popt
 @bp.route('/explore/place')
 def explore():
-    return redirect(url_for('place.place', dcid=request.args.get('dcid')),
+    return redirect('https://datacommons.org' +
+                    url_for('place.place', dcid=request.args.get('dcid')),
```

Comment:
did you want to add  _scheme='https' here?

Reply:
don't need to since we are specifying it. and it turns out _scheme seems to not compile / test for us. going to try without it and see if it works. i wish there was a better way to test this :(